spec/cspec/c/gen-config-thy.py

```diff
@@ -253,6 +271,10 @@ def write_config_thy(config_thy_path, config: Dict[str, str]):
             path.realpath(__file__))
         )
 
+        if physBase:
+            f.write('definition physBase :: machine_word where\n')
```

Comment:
do we want to spit out some comment here in anticipation of future victims?

Reply:
It's a generated file, so I don't think we want content comments in there. It'd be weird to have them in the generator. But we could add add something that points to where it's coming from, which also makes it less likely for explanations  to diverge.

Reply:
Have added a comment that this is `PHYS_BASE_RAW` and included the header file as dependency in the "generated" comment at the beginning of the file.